new-dti-website/src/app/page.tsx

```diff
@@ -96,7 +96,7 @@ const Home: React.FC = () => {
         <div className="flex flex-row grow h-full justify-evenly lg:gap-24 items-center lg:px-24 md:px-10 xs:px-4 mt-5">
           <div className="flex flex-col md:gap-8 xs:gap-4 xs:w-full lg:w-5/12">
             <h2 className="text-white md:text-[40px] xs:text-[28px] z-10 font-medium">
-              Cornell Digital Tech & Innovation Project Team
+              Building the Future of Tech @ Cornell
```

Comment:
Nice, this header is so much better now. A few nits:


On large screens, "@ Cornell" is a bit weird on its own separate line. Can we force a line break so that "of tech @ Cornell" is always together on the same line? I think it looks more visually balanced:

|Before|After|
|-|-|
|![Screenshot 2024-11-28 at 8 10 19 PM](https://github.com/user-attachments/assets/c8ad634b-5695-4a8f-998c-7a2324192c00)|![Screenshot 2024-11-28 at 8 11 06 PM](https://github.com/user-attachments/assets/53514b21-1196-4bc8-9be1-910b7af63eb4)|

Reply:
On smaller/mobile devices as well – when we resize the screen, the text wrapping is a bit unpredictable and it sometimes creates typographic widows where there's just a single word at the start of the new line.

Ideally, I think it'd be nice if we force "of Tech @ Cornell" to always be stuck together 

![Screenshot 2024-11-28 at 8 12 36 PM](https://github.com/user-attachments/assets/f5757370-e5a4-4ba6-af74-a536f37b20a6)